### DISCUSSION TITLE
Suggestion Order/Sorting

### DISCUSSION THREAD

**Discussion by ammodev** [redhat-developer/vscode-java#3135] [Q&A]
Hello since some time (I cannot realy define when it started) the suggestion order of IntelliSense seems to prefer the functions with the most arguments. I didn't have this before and it is kind of annoying as these are the methods I'm least using. 

Is there some option to get back to the original state of having the list turned around?
Current Behavior: https://i.imgur.com/UD7J5VZ.png - Most arguments function at the top
Wanted Behavior: https://i.imgur.com/zadGO1a.png - Have the suggestion list start with the bottom suggestion

**Comment by izderadicka** [comment]
Agree, I also consider this quite annoying - normally I prefer method with least parameters.  
Consider for instance testing with assertEquals,  where there are tons of overloads, but usually I just need one with 2 parameters.

I was trying to find some option in preferences but did not find anything that can change order of suggestions.

**Comment by fbricon** [comment]
you can try the latest pre-release, by default no parameters will be injected on completion, see https://github.com/redhat-developer/vscode-java/issues/2903#issuecomment-1594285429

**Reply by ammodev** [reply]
This fixes the issue of not having to delete every argument every time the method is completed. The list still seems to be turned around. Is there any fix for that?

**Reply by ammodev** [reply]
Any updates? @fbricon

**Comment by charles-wangkai** [comment]
I created an issue https://github.com/redhat-developer/vscode-java/issues/3206 for the same problem but haven't received a response.
